.ci/magic-modules/get_downstream_prs.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+import functools
+import os
+import re
+import sys
+from github import Github
+
+if __name__ == '__main__':
+  g = Github(os.environ.get('GH_TOKEN'))
+  all_prs = functools.reduce(lambda x,y: x+re.findall(r'\ndepends: (https://github.com/.*)', y.body),
+      g.get_repo('GoogleCloudPlatform/magic-modules').get_pull(int(sys.argv[1])).get_issue_comments(), [])
```

Comment:
I don't expect pagination to actually be an issue, but do you think it's worth it to just handle it now so it doesn't bite us later?